django/core/management/base.py

```diff
@@ -255,6 +255,7 @@ def create_parser(self, prog_name, subcommand):
             formatter_class=DjangoHelpFormatter,
             missing_args_message=getattr(self, 'missing_args_message', None),
             called_from_command_line=getattr(self, '_called_from_command_line', None),
+            **kwargs
```

Comment:
I just realized that this will throw a TypeError if anyone tries to override any of the kwargs specified above (for example `formatter_class`. Perhaps we should modify this to use a dictionary update as it was before?

Reply:
My thought was to wait and see if a use case was presented before adding that.